src/java/org/apache/cassandra/index/sai/disk/v1/SegmentBuilder.java

```diff
@@ -367,19 +382,19 @@ public boolean isEmpty()
         }
 
         @Override
-        protected long addInternal(ByteBuffer term, int segmentRowId)
+        protected long addInternal(List<ByteBuffer> terms, int segmentRowId)
         {
-            return graphIndex.add(term, segmentRowId);
+            return graphIndex.add(terms.get(0), segmentRowId);
```

Comment:
Seems like this could get an `assert terms.size() == 1;` too, since we're doing it in other peer methods.